test/test_nn.py

```diff
@@ -10962,6 +10962,13 @@ def test_channel_shuffle(self):
         y = y.contiguous(memory_format=torch.contiguous_format)
         self.assertEqual(y, y_ref)
 
+    def test_channel_shuffle_runtime_errors(self):
```

Comment:
One small change, can we add it under `class TestNNDeviceType` and update `torch.rand` to take device, so that this test would also run on CUDA

Thanks!